src/reactions/ReactionList.js

```diff
@@ -21,8 +21,12 @@ export default class ReactionList extends React.PureComponent {
   render() {
     const { messageId, reactions, selfEmail } = this.props;
 
-    if (reactions.length === 0) {
-      return null;
+    if( typeof reactions !== 'undefined' ) { 
```

Comment:
I suspect your server does not support reactions? Haven't thought of that.
The only change needed is from:
`if (reactions.length === 0) {`
to
`if (!reactions || reactions.length === 0) {`